opencti-platform/opencti-front/src/private/components/common/files/FileExportViewer.tsx

```diff
@@ -48,6 +48,7 @@ FileExportViewerComponentProps
       subscription.unsubscribe();
     };
   }, []);
+
```

Comment:
if there is only this empty line as a change in this file, maybe remove it ?

Reply:
Empty lines make the code more readable :)

Reply:
Yep we thought about removing this file from the changes but the reasoning was that's very frustrating to have one bloc of code without any spaces, which is common in our codebase, so we decided to keep it because one empty line cannot hurt us while reading the code 😅